### DISCUSSION TITLE
Account restricted

### DISCUSSION THREAD

**Discussion by VinofSteel** [codewars/codewars.com#2854] [Q&A]
I don't know if this is the right place to write, but CodeWars forum gives me a message that it's deprecated, so here we go 

My codewars account got restricted, I don't know if it's because I did too many solutions in a short time frame, but I lost access to an older account (my password was unsafe and I figure someone messed with it) and created a new one. Since I'm quite more experienced at codewars right now than I was before, I got a lot of points very quickly, which I assume is what got my account restricted, but that makes no sense to me. 

I didn't cheat and I can explain and show all of my code, that I developed myself. My account is currently named Vintner.

I'd really like to get my restriction lifted and understand why this is happening, cause I don't want to just create another account, do all the work to get points and then get restricted again.

Thanks in advance.

**Comment by Blind4Basics** [comment]
* Would you by any chance still have the link to your first account?
* when did it get ""corrupted""?
* ... do you really believe that people will mess your CW account...? (that's kinda funny, I have to say)

**Reply by VinofSteel** [reply]
Ok, a bit of personal backstory on this.

On my old workplace (which I worked at as a front-end developer) we had some friendly codewars competitions of people who could solve a Kata first, or who would get a certain score first, so on, so forth. I had some very unpleasant interactions with one of the staff that was higher on the hierarchy than me and I have reason to believe this person is the one that invaded my old account, which was named VinofSteel. I've looked the name up on codewars to find the link, but I couldn't find it, figure this person changed it. I say it was messed with because I, being dumb, used the same password on lots of different accounts, and my emails and other personal stuffed have suffered invasion attempts on these last few weeks. Since I have reason to believe this person at work was the one doing this, it makes sense to me that they also invaded and messed with my CW account.

I understand this is hard to believe, specially figuring that I have no way to show links to the old account, but, as I said, I can verify the solutions to my Kata answers. I could just create another account and re-do everything (even if it takes time), but I prefer to use the proper channels. If possible, I'd like to also understand the criteria that flagged me as a cheater, because I literally wrote all of my code without copying anything even once.

**Comment by Blind4Basics** [comment]
You didn't give a time frame, there. Am I correct, when interpreting your message, that this happened in the last few days? Maybe weeks?

**Reply by VinofSteel** [reply]
Yes, the invasions begin happening around 3 weeks ago. I just tried accessing my CodeWars account about a week, a week and a half ago, that's when I figured this happened and created the new one

**Reply by Blind4Basics** [reply]
OK, you just confirmed you are a copycat anyway. 

> I literally wrote all of my code without copying anything even once.

That's not true, you used copies _months_ ago on that first account of yours (yes, we found it). Your account will stay locked. You ofc can delete it and start fresh, but be aware that any suspicious activity will lead to a new restriction.
